cmd/minikube/cmd/delete.go

```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/minikube/pkg/minikube/localpath"
 	"k8s.io/minikube/pkg/minikube/machine"
 	"k8s.io/minikube/pkg/minikube/out"
+	"k8s.io/minikube/pkg/minikube/out/register"
```

Comment:
irrelelvent, but I wonder if there is a better name or better place for this package ?

something like status or step ... not sure

Reply:
Agreed.